metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -141,8 +141,9 @@ def kill(ctx, run_id, user, my_runs):
 @click.option("--max-swap", help="Max Swap requirement for AWS Batch.")
 @click.option("--swappiness", help="Swappiness requirement for AWS Batch.")
 # TODO: Maybe remove it altogether since it's not used here
-@click.option("--ubf-context", default=None, type=click.Choice([None]))
+@click.option("--ubf-context", default=None, type=click.Choice([None, "ubf_control"]))
 @click.option("--host-volumes", multiple=True)
+@click.option("--num-parallel", type=int)
```

Comment:
Should we add a default?

Reply:
I think None is the proper default here.

Reply:
Sorry, I should have been clearer: I was suggesting making these options consistent across CLIs (both for this and the ubf-context above). The `step` command in the main CLI has the same arguments but different types/values which, although it should work fine, it may be preferable to make them consistent.

Reply:
Ah got it, makes sense.